lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -577,26 +603,40 @@ monitorMetadata tr gp fetchMetadata DBLayer{..} = forever $ do
         f = unActiveSlotCoefficient (getActiveSlotCoefficient gp)
 
 data StakePoolLog
-    = MsgFollow FollowLog
+    = MsgDb PoolDbLog
+    | MsgFollow FollowLog
     | MsgStartMonitoring [BlockHeader]
     | MsgHaltMonitoring
     | MsgCrashMonitoring
     | MsgRollingBackTo SlotNo
+    | MsgStakePoolGarbageCollection PoolGarbageCollectionInfo BracketLog
     | MsgStakePoolRegistration PoolRegistrationCertificate
     | MsgStakePoolRetirement PoolRetirementCertificate
     | MsgErrProduction ErrPointAlreadyExists
     | MsgFetchPoolMetadata StakePoolMetadataFetchLog
     | MsgFetchTakeBreak Int
     deriving (Show, Eq)
 
+data PoolGarbageCollectionInfo = PoolGarbageCollectionInfo
+    { currentEpoch :: EpochNo
+        -- ^ The current epoch at the point in time the garbage collector
+        -- was invoked.
+    , removalEpoch :: EpochNo
+        -- ^ The removal epoch: the garbage collector will remove all pools
+        -- that retired on or before this epoch.
+    }
+    deriving (Eq, Show)
+
 instance HasPrivacyAnnotation StakePoolLog
 instance HasSeverityAnnotation StakePoolLog where
     getSeverityAnnotation = \case
+        MsgDb e -> getSeverityAnnotation e
         MsgFollow e -> getSeverityAnnotation e
         MsgStartMonitoring{} -> Info
         MsgHaltMonitoring{} -> Info
         MsgCrashMonitoring{} -> Error
         MsgRollingBackTo{} -> Info
+        MsgStakePoolGarbageCollection{} -> Info
```

Comment:
This should probably be lowered to `Debug` in order to avoid spamming the log. In particular, we don't need to see any informational messages when there are no pools to garbage collect.

Reply:
> This should probably be lowered to `Debug` in order to avoid spamming the log. In particular, we don't need to see any informational messages when there are no pools to garbage collect.

Fixed in b1e4d1506c15f1be05fc1ad2416badee69a47529.